dream_layer_backend/dream_layer.py

```
@@ -483,10 +500,199 @@ def get_controlnet_models_endpoint():
+                        current_progress.update({
+                            "percent": 100,
+                            "status": "complete",
+                            "message": "Complete!"
+                        })
+                    elif current_progress["percent"] == 0:
+                        current_progress.update({
+                            "percent": 0,
+                            "status": "idle",
+                            "message": "Ready"
+                        })
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)